src/otx/core/model/entity/classification.py

```diff
@@ -46,7 +46,8 @@ def device(self) -> device:
             else:
                 return buf.device
 
-    return build_mm_model(config, MODELS, load_from)
+    classification_layers = get_classification_layers(config, MODELS, "model.")
+    return build_mm_model(config, MODELS, load_from), classification_layers
```

Comment:
Is this possible although the return type is `nn.Module` at the line 32?

Reply:
Changed by 
b095a26 It seems mypy do not test return type strictly